### PR TITLE
inventories: refactor SEAPATH inventories samples

### PR DESCRIPTION
**Pull request by dupremathieu**
Regroup variables by theme and add comments.

### CODE REVIEW

File: inventories/seapath_standalone_definition_example.yml [hunk 1]
```diff
@@ -4,46 +4,61 @@ all:
...
-        dns_server: 10.10.2.1   # Not valid, just for Ansible
-        gateway_addr: 10.0.0.1 # Not valid, just for Ansible
-        apply_network_config: true
-        ntp_primary_server: "ntp.midway.ovh"
-        ntp_secondary_server: "ntp.unice.fr"
-        ansible_remote_tmp: /tmp/.ansible/tmp
-        syslog_server_ip: "10.0.0.10"
-        logstash_server_ip: "10.0.0.11"
-        workqueuemask: "1001"
-        irqmask: "ffeffe"
```
Comment (eroussy) [inline inventories/seapath_standalone_definition_example.yml:34]: This line is not added anywhere in the file, I think you forget it

File: inventories/seapath_standalone_definition_example.yml [hunk 2]
```diff
@@ -4,46 +4,61 @@ all:
...
+
+                    # Optional list variable to change apt repositories. All repositories will be overrides.
+                    # If defined as empty list [], all apt repositories will be removed
+                    apt_repo:
+                        - http://ftp.fr.debian.org/debian bullseye main contrib non-free
+                        - http://security.debian.org/debian-security bullseye-security main contrib non-free
+                        - http://ftp.fr.debian.org/debian bullseye-backports main contrib non-free
```
Comment (eroussy) [inline inventories/seapath_standalone_definition_example.yml:62]: why add this line ?

Comment (dupremathieu) [reply inventories/seapath_standalone_definition_example.yml:62]: I reuse the apt_repo from the previous inventory.
The kernel we use came from bullseye-backports. We need this repo if we want update it.

File: inventories/seapath_standalone_definition_example.yml (hunk 1 above)
Comment (dupremathieu) [reply inventories/seapath_standalone_definition_example.yml:34]: Done

**Review by eroussy — APPROVED** [review]

File: inventories/seapath_cluster_definition_example.yml [hunk 3]
```diff
@@ -6,124 +6,169 @@ all:
                 hypervisors:
                     hosts:
                         node1:
+                            # ansbile variables
```
Comment (insatomcat) [inline inventories/seapath_cluster_definition_example.yml:9]: ansbile --> ansible

**Pull request merged**